qucs/configure.ac

```diff
@@ -236,7 +236,7 @@ case $host_os in
 
     dnl Enable C++11 support
     CXXFLAGS="$CXXFLAGS -std=c++0x"
-    PKG_CHECK_MODULES([QTest], [${qt_name}Test ])
+    PKG_CHECK_MODULES([QTest], [${qt_name}Test])
```

Comment:
by the way, any idea why this works with and without the space on my system and on Travis too?
Here everything also works fine also if I remove this line, so in which case is it needed??

Trying to understand what `PKG_CHECK_MODULES` does, I saw that "[this guy](http://tirania.org/blog/archive/2012/Oct-20.html)" and others discourage its usage; FWIW, I knowing nothing about it.